repos/system_upgrade/common/actors/systemd/checksystemdservicetasks/actor.py

```diff
@@ -0,0 +1,24 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import checksystemdservicetasks
+from leapp.models import SystemdServicesTasks
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, TargetTransactionChecksPhaseTag
+
+
+class CheckSystemdServicesTasks(Actor):
+    """
+    Inhibits upgrade if SystemdServicesTasks tasks are in conflict
+
+    There is possibility, that SystemdServicesTasks messages with conflicting
+    requested service states could be produced. For example a service is
+    requested to be both enabled and disabled. This actor inhibits upgrade in
+    such cases.
```

Comment:
I would add a note so other people are aware about expectations.

```suggestion
    such cases.
    
    Note: We expect that SystemdServicesTasks could be produced even after the TargetTransactionChecksPhase
    (e.g. during the ApplicationPhase). The purpose of the actor is reduce such a collisions in case we already
    know detect them. In case of conflicts made by lately produced messages we just log such collisions and
    such services will end up disabled.
```